packages/ipfs-core-utils/src/files/normalise-input/utils.js

```diff
@@ -12,10 +10,12 @@ function isBytes (obj) {
 
 /**
  * @param {any} obj
- * @returns {obj is Blob}
+ * @returns {obj is globalThis.Blob}
  */
 function isBlob (obj) {
-  return typeof Blob !== 'undefined' && obj instanceof Blob
+  return obj.constructor &&
```

Comment:
@achingbrain what are your feelings on using here `@web-std/blob` ? That would work instead of "supporting browser only"